src/js/view/view-model.ts

```diff
@@ -111,11 +123,11 @@ export default class ViewModel extends PlayerViewModel {
         }, this);
     }
 
-    get player() {
+    get player(): PlayerViewModel {
         return this._playerViewModel;
     }
 
-    set instreamModel(instreamModel) {
+    set instreamModel(instreamModel: Model) {
```

Comment:
It's odd that this is a public setter. It's only used above in the change event handler so really this should be scoped there, but no need to change it now.